### PR TITLE
Add translation about/governance in Spanish

### PR DESCRIPTION
**Pull request by Beor18**
Hello community,

I bring you another contribution in the Spanish translation

Thank you very much and I hope for comments.

Best regards

### CODE REVIEW

**Comment by fhemberger** [comment]
@Beor18 Hi, thanks for the update. Could you please rebase your changes from master and fix the merge conflicts?

/cc @nodejs/nodejs-es Please review.

**Comment by Beor18** [comment]
> @Beor18 Hi, thanks for the update. Could you please rebase your changes from master and fix the merge conflicts?
> 
> /cc @nodejs/nodejs-es Please review.

Hello @fhemberger, have now if it's okay.
Excuse me as I am new contributing by there I can make some mistake and it's good to let me know jajaja

Best regards!

**Comment by ghost** [comment]
@nodejs/nodejs-es：Sorry but it's been a long time, anyone free can have a review of that so as to merge it quickly :)

File: locale/es/about/governance.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Proceso de búsqueda de consenso
+
+El proyecto Node.js sigue un modelo de [Búsqueda de Consenso][] en la toma de decisiones.
+
+## Colaboradores
+
+El repositorio GitHub de [nodejs/node][] es mantenido por los Colaboradores que son agregados por el Comité Directivo Técnico ([TSC][]) de forma continua.
+
+Las personas que hacen contribuciones importantes y valiosas se convierten en Colaboradores y se les otorga acceso comprometido al proyecto. Estas personas están identificadas por el TSC y su nominación se discute con los Colaboradores existentes.
```
Comment (academo) [inline locale/es/about/governance.md]: "access comprometido" no tiene sentido aqui, siendo un termino tan tecnico seria mejor usar "commit-access" (con comillas) aqui?

Comment (Beor18) [reply locale/es/about/governance.md]: entonces seria

_se le otorga un "commit-access" al proyecto._ ???

Comment (edsadr) [reply locale/es/about/governance.md]: es mejor una traducción menos literal usar por ejemplo `permisos de escritura` estaría bien.

**Review by edsadr — CHANGES_REQUESTED** [review]

File: locale/es/about/governance.md (hunk 1 above)
Comment (Beor18) [reply locale/es/about/governance.md]: perfecto!

**Review by edsadr — APPROVED** [review]

**Comment by ghost** [comment]
@Beor18：Hi,

Since your changes should be limited to 'about/governance', so for other changes you don't need to submit. You can only submit what changes you have :)

Suggestions:
1)  Save your changes into 'Stash' first.
2)  You can just use `git pull --rebase origin master` to update your own master branch when it's checked out.
3)  Solve all kinds of conflicts...ect.
4)  Apply the changes from your Slash of Git into the current master, and make a force push.

Hope this helps!

**Review by vrunoa — APPROVED** [review]

**Comment by ghost** [comment]
@Beor18：Thank you!

**Pull request merged**